arkane/gaussian.py

```diff
@@ -274,7 +274,7 @@ def load_energy(self, zpe_scale_factor=1.):
                 if 'SCF Done:' in line:
                     e_elect = float(line.split()[4]) * constants.E_h * constants.Na
                     elect_energy_source = 'SCF'
-                elif ' E2(' in line:
+                elif ' E2(' in line and ' E(' in line:
```

Comment:
I am probably missing something. What line in ethyl_cbsqb3.log should this be parsing?

When I type `grep ' E2(' ethyl_cbsqb3.log | grep ' E('`, I don't find any matching lines.

The unit tests are passing, though I don't understand why.

Reply:
Yes, that how it should be. In my previous commit, I only put " E2("  and it found a match at the end of the summary gaussian print. That's why the unit test failed.  " E2("  and "E(" does not appear together in the cbs-qb3 calculation and it will correctly parse the " cbs-qb3 energy" , but it will appear in the double-hybrid calculations.